packages/react-html/src/manager.ts

```diff
@@ -130,3 +130,30 @@ export class HtmlManager {
     }
   }
 }
+
+function filterMetas(metas: React.HTMLProps<HTMLMetaElement>[]) {
```

Comment:
nitpik: maybe call this `removeDuplicate` ?

also since the order here is important, maybe reverse it again before return so order maintain the same.